guided-match/dml-versions/v021.sql

```diff
@@ -0,0 +1,108 @@
+('Ppm - page-per-minute speed','Print and Digital', 'Provides access to multifunctional devices, managed print services, technical resources, digital workflow, cloud solutions for digital transition and print consultancy services.'),
+('Page per minute','Print and Digital', 'Provides access to multifunctional devices, managed print services, technical resources, digital workflow, cloud solutions for digital transition and print consultancy services.'),
+('Print management','Print and Digital', 'Provides access to multifunctional devices, managed print services, technical resources, digital workflow, cloud solutions for digital transition and print consultancy services.'),
+('Print room','Print and Digital', 'Provides access to multifunctional devices, managed print services, technical resources, digital workflow, cloud solutions for digital transition and print consultancy services.'),
+('Printed','Print and Digital', 'Provides access to multifunctional devices, managed print services, technical resources, digital workflow, cloud solutions for digital transition and print consultancy services.'),
+('Printers','Print and Digital', 'Provides access to multifunctional devices, managed print services, technical resources, digital workflow, cloud solutions for digital transition and print consultancy services.'),
+('Printing','Print and Digital', 'Provides access to multifunctional devices, managed print services, technical resources, digital workflow, cloud solutions for digital transition and print consultancy services.'),
+('Process efficiencies','Print and Digital', 'Provides access to multifunctional devices, managed print services, technical resources, digital workflow, cloud solutions for digital transition and print consultancy services.'),
+('Recommendations','Print and Digital', 'Provides access to multifunctional devices, managed print services, technical resources, digital workflow, cloud solutions for digital transition and print consultancy services.'),
+('Strategic change','Print and Digital', 'Provides access to multifunctional devices, managed print services, technical resources, digital workflow, cloud solutions for digital transition and print consultancy services.'),
+('Tech','Print and Digital', 'Provides access to multifunctional devices, managed print services, technical resources, digital workflow, cloud solutions for digital transition and print consultancy services.'),
+('Technology','Print and Digital', 'Provides access to multifunctional devices, managed print services, technical resources, digital workflow, cloud solutions for digital transition and print consultancy services.'),
+('Technology products','Print and Digital', 'Provides access to multifunctional devices, managed print services, technical resources, digital workflow, cloud solutions for digital transition and print consultancy services.');
+
+
```

Comment:
Ordering is incorrect after this point. You should be 

1. adding **temp_domain_values** keywords to **search_terms**
2. map new keywords with GM Lite
3. add existing keyword to **temp_domain_values**
4. then you can add everything from **temp_domain_values** to **search_domains**

Take a look at v017.sql and follow the structure

Reply:
Makes sense, thanks for the feedback Chee!